sklearn/cluster/tests/test_optics.py

```diff
@@ -197,7 +197,7 @@ def test_minimum_number_of_sample_check():
 
     # Compute OPTICS
     X = [[1, 1]]
-    clust = OPTICS(max_eps=5.0 * 0.3, min_samples=10, min_cluster_size=1)
+    clust = OPTICS(max_eps=5.0 * 0.3, min_samples=10, min_cluster_size=1.0)
```

Comment:
Technically this breaks backward compatibility, but the docs says `int > 1`, so I'm okay with being strict here.

https://github.com/scikit-learn/scikit-learn/blob/fd9bff1fd20b129f516d4675b24aeacfdd3bac56/sklearn/cluster/_optics.py#L127